rsa/_compat.py

```diff
@@ -26,6 +26,8 @@
 MAX_INT32 = (1 << 31) - 1
 MAX_INT16 = (1 << 15) - 1
 
+PY3 = sys.version_info[0] >= 3
```

Comment:
Yes, that would be more accurate. But as I see it, if Python 4 is released tomorrow, I would prefer to have py3's configurations rather than py2's (unless until a new `PY4` constant is added).

With that in mind, what do you think about using `PY2` instead?